internal/configs/configmaps_test.go

```diff
@@ -285,6 +285,106 @@ func TestParseConfigMapAccessLogDefault(t *testing.T) {
 	}
 }
 
+func TestParseMGMTConfigMapNil(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		configMap *v1.ConfigMap
+		want      *MGMTConfigParams
+		msg       string
+	}{
+		{
+			configMap: &v1.ConfigMap{
+				Data: map[string]string{
+					"license-token-secret-name": "",
+				},
+			},
+			want: nil,
+			msg:  "Must have license-token-secret-name",
+		},
+		{
+			configMap: &v1.ConfigMap{
+				Data: map[string]string{},
+			},
+			want: nil,
+			msg:  "Must have license-token-secret-name key",
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.msg, func(t *testing.T) {
+			result, _, err := ParseMGMTConfigMap(context.Background(), test.configMap, makeEventLogger())
+			if err == nil {
+				t.Fatal(err)
+			}
+			if result.Secrets.License != test.want.Secrets.License {
```

Comment:
var `want` here is always `nil`, correct? Am I right that the table tests here contain values that should make the `ParseMGMTConfigMap` func return error? If that's true, other returned values are irrelevant.

Reply:
Yes, the only way we actually get an error from `ParseMGMTConfigMap` is from the license

Reply:
So, if the test inspects the `License` then error should be not `nil`. In this example if the `err` == `nil` the test fails.

Reply:
We have refactored the test, can we have another review on that please